torch/_refs/__init__.py

```diff
@@ -2200,16 +2206,21 @@ def _make_copy_from_view(fn):
     """
     Given a view function (e.g. torch.diagonal) generates its copy variant (e.g. torch.diagonal_copy)
     """
-    name = fn.__name__
-    fn = out_wrapper()(fn)
+    aten_fn = getattr(aten, fn.__name__)
+    fn = out_wrapper(annotations=dict(fn.__annotations__))(aten_fn)
```

Comment:
I'm not thrilled by this new param to `out_wrapper`. Can't you simply do
`_fn.__annotations__ = fn.__annotations__` after we set `_fn.__name__`?

Reply:
DOH! Why didn't I think of that?

Updated.